lib/runtime/life/resolver.go

```diff
@@ -58,15 +68,65 @@ func (r *Resolver) ResolveFunc(module, field string) exec.FunctionImport {
 		case "ext_storage_root_version_1":
 			return ext_storage_root_version_1
 		case "ext_storage_changes_root_version_1":
+			// todo doesn't have rtm test
 			return ext_storage_changes_root_version_1
 		case "ext_crypto_start_batch_verify_version_1":
+			// todo how to implement this?
 			return ext_crypto_start_batch_verify_version_1
 		case "ext_crypto_finish_batch_verify_version_1":
+			// todo how to implement this?
 			return ext_crypto_finish_batch_verify_version_1
 		case "ext_offchain_index_set_version_1":
+			// todo how to implement this?
```

Comment:
I think if there's no corresponding test in `wasmer/imports_test.go` at the moment, you can leave it. some of the tests in `exports_test` should cover these

Reply:
Ok, removed todos.